### DISCUSSION TITLE
File Explorer keeps starting in own process (with Windows 11 style)

### DISCUSSION THREAD

**Discussion by LarsErikOsterud** [valinet/ExplorerPatcher#1640] [Q&A]
File Explorer keeps starting in own process (with Windows 11 style).

I have set all registry keys but after a shile Windows 11 keeps changing one are more back back.
Have to remove all write access from the key to keep the setting (and the classic file explorer).
Any ideas?

"[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer]" "DekstopProcess" = 0"
"[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced]" "SeparateProcess" = "0" 
"[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer]" ""DesktopProcess" = "0" and
"[HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced]" ""SeparateProcess" = "0"

**Comment by pyrates999** [comment]
Please post windows version and build.
Please post explorer patcher version.

**Comment by Wolf-Shadow777** [comment]
I have same issue using ep with windows 11 Pro 64 bit build 23493. explorer keeps rebooting every couple of minutes on its own. uninstall and reinstall of ep has no effect. Left uninstalled until issue is resolved. Using the latest release 
Version: 22621.1555.55.2

**Comment by pyrates999** [comment]
Preview builds are not supported because symbols are not available to download for them.

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by LarsErikOsterud** [comment]
This has happend for me for quite a while with both pre-releases end official releases.
And it just seems like somthing changes the registry setting to enable separate processen.

Current versions are:   Windows 11:  22H2 22621.1992  ExplorerPatcher:  22621.1555.55.2

**Comment by pyrates999** [comment]
I would suggest that you uninstall explorer patcher and change those settings in the registry.  Then monitor them to see if they change.

**Comment by MadManMoon** [comment]
Is this related to the looping explorer thing? As if so I've been seeing this too (_mistakenly [raised issue](https://github.com/valinet/ExplorerPatcher/issues/1652) - sorry, pyrates999!_) on my work PC pretty much rendering it unusable until uninstall.

Whether it is that or not, that particular thing comes on when Windows has pushed a feature update (_which I'd thought I'd disabled - but I am stupidly on Insider_) which changes whatever it changes.

The first true restart after the update, and you just can't do anything with your Windows system as explorer refuses to start up correctly.

**Reply by pyrates999** [reply]
If you're running an insider build, those are not supported because microsoft doesn't make symbols available to download for them: https://github.com/valinet/ExplorerPatcher/wiki/Symbols
